docs/index.html

```diff
@@ -45,6 +55,7 @@
     </script>
     <script type="text/javascript">
       const options = {
+        highlightLines: true
```

Comment:
This allows for highlighting lines of code blocks within \`\`\` sections. For eg, on slide 23:

---

![highlight](https://user-images.githubusercontent.com/2654835/71649916-6dc59200-2cc7-11ea-829e-c2ef39fa8db1.png)